scb-scanprocesses/combined-nmap-ssh-process/src/main/resources/forms/nmap-ssh/configure-target.html

```diff
@@ -0,0 +1,131 @@
+<!--
+  ~ /*
+  ~ * SecureCodeBox (SCB)
+  ~ * Copyright 2015-2018 iteratec GmbH
+  ~ *
+  ~ * Licensed under the Apache License, Version 2.0 (the "License");
+  ~ * you may not use this file except in compliance with the License.
+  ~ * You may obtain a copy of the License at
+  ~ *
+  ~ * 	http://www.apache.org/licenses/LICENSE-2.0
+  ~ *
+  ~ * Unless required by applicable law or agreed to in writing, software
+  ~ * distributed under the License is distributed on an "AS IS" BASIS,
+  ~ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ * See the License for the specific language governing permissions and
+  ~ * limitations under the License.
+  ~ */
+  -->
+
+<h2>Please configure the Port Scan</h2>
+
+<form role="form" name="configure-target">
+  <script cam-script type="text/form-script">
+    camForm.on('form-loaded', function () {
+        camForm.variableManager.fetchVariable('DEFAULT_CONTEXT');
+        camForm.variableManager.fetchVariable('DEFAULT_TARGET_NAME');
+        camForm.variableManager.fetchVariable('DEFAULT_TARGET_LOCATION');
+    });
+
+    camForm.on('variables-fetched', function () {
+        $scope.context = camForm.variableManager.variableValue('DEFAULT_CONTEXT');
+
+        // What minimal --version-intensity includes ssh services? --version-all is not needed if it's known
+        $scope.targetList = [{
+            name: camForm.variableManager.variableValue('DEFAULT_TARGET_NAME'),
+            location: camForm.variableManager.variableValue('DEFAULT_TARGET_LOCATION'),
+            attributes: {
+                }
+        }];
+
+        $scope.checkForEnter = function ($event) {
+            if ($event.key === 'Enter') {
+                $event.stopPropagation();
+                $event.preventDefault();
+            }
+        };
```

Comment:
This function isn't needed anymore as it was only used to handle the multi target form stuff.
```suggestion
```